packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "20.0.0",
+  "version": "21.0.0",
```

Comment:
If core is being bumped in this PR, we should create a release after this PR is merged.